workflow/deploy.md

```diff
@@ -51,16 +51,12 @@ In order to deploy to Kubernetes you will need to create 2 Jenkins jobs on the w
 
 Update the **apply** job:
 
-- Go to the [configure page](https://jenkins.canonical.com/webteam/job/apply-production-deployment-configs/configure)
-- Add these lines:
+- If you are going to deploy a new site, check that the file `/sites/<DNS_OF_THE_WEBSITE>.yaml` exists in the [deployment configs repository](https://github.com/canonical-web-and-design/deployment-configs)
```

Comment:
I think `DOMAIN_NAME` is a better than `DNS_OF_THE_WEBSITE` (and I prefer `{}` to `<>` - as the latter can easily accidentally get interpreted as an HTML tag in Markdown)

```suggestion
- If you are going to deploy a new site, check that the file `/sites/{DOMAIN_NAME}.yaml` exists in the [deployment configs repository](https://github.com/canonical-web-and-design/deployment-configs)
```